data/src/main/java/co/nimblehq/data/service/response/ErrorResponse.kt

```diff
@@ -1,8 +1,7 @@
 package co.nimblehq.data.service.response
 
-import com.google.gson.annotations.SerializedName
+import com.squareup.moshi.Json
 
 data class ErrorResponse(
-    @SerializedName("message")
-    val message: String
+    @Json(name = "message") val message: String
```

Comment:
Should we keep the old format with annotation and val separate line? 🤔

Reply:
hmm, I think in new projects we implement them in one-line like DeeMoney so I guess it is fine to keep. What do you think? 🤔

Reply:
I'm not a fan of keeping all things in just 1 line 🤪 
for example
```
@Inject
lateinit var navigator: MainNavigator
```
or
```
@Json(name = "message") 
val message: String
@Json(name = "error_code") 
val errorCode: Int
```
looks cleaner and easy to look for all fields vertically